plugins/gradle/graphql-java-codegen-gradle-plugin/src/main/java/io/github/kobylynskyi/graphql/codegen/gradle/GraphQLCodegenGradleTask.java

```diff
@@ -121,6 +122,7 @@ public class GraphQLCodegenGradleTask extends DefaultTask implements GraphQLCode
     private String unknownFieldsPropertyName = MappingConfigConstants.DEFAULT_UNKNOWN_FIELDS_PROPERTY_NAME;
 
     private Boolean skip = false;
+    private Boolean skipSchemaSizeLimit = true;
```

Comment:
Please use `MappingConfigConstants.DEFAULT_` for the default value of this parameter (for consistency purposes).